src/main/scala/ai/privado/audit/DataElementDiscovery.scala

```diff
@@ -162,6 +164,39 @@ object DataElementDiscovery {
     collectionMethodInfoMap.toMap
   }
 
+  def getFileScore(absoluteFileName: String, xtocpg: Try[Cpg]): String = {
```

Comment:
Create unit test Case for this Function

Reply:
Added here: https://github.com/Privado-Inc/privado-core/pull/606/commits/385b691b9e8d0dfbf6c70cd1a9671bb76d4b26e7